opm/simulators/wells/WellGroupHelpers.cpp

```diff
@@ -403,6 +403,7 @@ updateGroupTargetReduction(const Group& group,
                            std::vector<Scalar>& groupTargetReduction)
 {
     OPM_TIMEFUNCTION();
+    bool has_choke = schedule.hasChoke(group.name(), reportStepIdx);
```

Comment:
`const bool` here. And also, it is good to define the variable as close as possible to the actual usage. Since the usage is within a for loop, you can define it right before the for loop that actually use the variable to avoid repeated evaluation.